health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreenParent/ImpelComponentWrapper.js

```
@@ -0,0 +1,95 @@
+import AppLocalisationWrapper from "./AppLocalisationWrapper";
+
+function ImpelComponentWrapper({ variant, screenConfig, submit, back, showBack, parentDispatch, ...props }) {
+  const MODULE_CONSTANTS = "HCM-ADMIN-CONSOLE";
+  const searchParams = new URLSearchParams(location.search);
+  const fieldMasterName = searchParams.get("fieldType");
+  const { isLoading: isLoadingAppConfigMdmsData, data: AppConfigMdmsData } = Digit.Hooks.useCustomMDMS(
+    Digit.ULBService.getCurrentTenantId(),
+    MODULE_CONSTANTS,
+    [{ name: fieldMasterName }],
+    {
+      cacheTime: Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)